source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -585,3 +585,7 @@
 		padding: var(--wp--preset--spacing--10) !important;
 	}
 }
+
+.is-style-list-long-items > li {
+	margin-bottom: var(--wp--preset--spacing--30);
```

Comment:
The regular paragraph spacing is [20px](https://github.com/WordPress/wporg-parent-2021/blob/2b7ee519917e808de71e17e424e3a29287497e70/source/wp-content/themes/wporg-parent-2021/theme.json#L791), so I think `--wp--preset--spacing--20` would be more consistent here. I see the design uses 30px between paragraphs, but that doesn't match the other site designs.

Reply:
Updated!